contracts/tokens/SecurityToken.sol

```diff
@@ -71,6 +63,15 @@ contract SecurityToken is ERC20, ReentrancyGuard, SecurityTokenStorage, IERC1594
     event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
     event TokenUpgraded(uint8 _major, uint8 _minor, uint8 _patch);
 
+    // Emit when Module get archived from the securityToken
+    // event ModuleArchived(uint8[] _types, address _module); Event emitted by the tokenLib.
+    // Emit when Module get unarchived from the securityToken
+    // event ModuleUnarchived(uint8[] _types, address _module); Event emitted by the tokenLib.
+    // Emit when Module get removed from the securityToken
+    // event ModuleRemoved(uint8[] _types, address _module); Event emitted by the tokenLib.
+    // Emit when the budget allocated to a module is changed
+    // event ModuleBudgetChanged(uint8[] _moduleTypes, address _module, uint256 _oldBudget, uint256 _budget); Event emitted by the tokenLib.
+
```

Comment:
Why did we commented them? AFAICT these events are emitted by the `TokenLib` library and we have to also declare them in the token contract to make them visible to Etherscan. isn't?

Reply:
> we have to also declare them in the token contract to make them visible to Etherscan. isn't?

I am not sure if that is the case. Have you tested? If it is the case, I'll add these back.

Reply:
yes, I am pretty sure about this.